src/components/CytoscapeGraph/graphs/GraphStyles.ts

```diff
@@ -33,7 +33,7 @@ const EdgeIconMTLS = icons.istio.mtls.ascii; // lock
 const EdgeIconDisabledMTLS = icons.istio.disabledMtls.ascii; // broken lock
 const EdgeTextOutlineColor = PfColors.White;
 const EdgeTextOutlineWidth = '1px';
-const EdgeTextFont = 'Verdana,Arial,Helvetica,sans-serif,FontAwesome,PatternFlyIcons-webfont';
+const EdgeTextFont = 'Verdana,Arial,Helvetica,sans-serif,FontAwesome,PatternFlyIcons-webfont,pficon';
```

Comment:
Maybe we can remove ",FontAwesome,PatternFlyIcons-webfont", I'll test it when I run the PR.

Reply:
@jshaughn I tested that out and it looks like you are correct we can remove that

![Kiali_Console](https://user-images.githubusercontent.com/1312165/76873746-6411ec00-682b-11ea-801f-7ed7b4fa5a07.png)

Reply:
If we change this we will want to get QE @hhovsepy to re-test just to be sure.

Reply:
I thought I had removed those two. They are removed in the last commit.
Both of you have tested, me and alberto. I think it is good to go.